go/enclave/components/block_processor.go

```diff
@@ -115,7 +115,7 @@ func (bp *l1BlockProcessor) tryAndInsertBlock(br *common.BlockAndReceipts) (*Blo
 		return nil, err
 	}
 	bp.logger.Trace("Block inserted successfully",
-		log.BlockHeightKey, block.NumberU64(), log.BlockHashKey, block.Hash(), "ingestionType", ingestionType)
```

Comment:
As I think Tudor pointed out, this was actually an issue in the `String()` method of `ChainFork`. I guess a nil pointer on line 188 of `go/common/types.go`. Could we maybe do the checks in there so it can't bite anyone else and we don't lose this useful context.